src/index.html

```diff
@@ -11,7 +11,193 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="field-set">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname"> Surname: </label>
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+            autocomplete="off"
+          >
+
+        </div>
```

Comment:
No need in empty lines between parent and child elements.
```suggestion
          >
        </div>
```